### PR TITLE
chore: create community and edit community tests updated

### PR DESCRIPTION
**Pull request by Valentina1133**
- Updated test for create community (more verifications added)
- Also edit_community updated
- Edit separately test deleted
- I didn't join create community and edit community test because it will be too long scenario in this case

https://ci.status.im/job/status-desktop/job/e2e/job/manual/1468/

### CODE REVIEW

File: tests/communities/test_communities_screens_overview.py [hunk 1]
```diff
@@ -17,7 +17,9 @@
 @pytest.mark.parametrize('params', [constants.community_params])
 def test_manage_community_screens_overview(main_screen: MainWindow, params):
     with step('Create community'):
-        main_screen.create_community(params)
```
Comment (anastasiyaig) [inline tests/communities/test_communities_screens_overview.py:20]: can you please fix the reference to test cases? https://github.com/status-im/desktop-qa-automation/blob/7547dcd830a8b44c7bc50080dfcb80fa76df391c/tests/communities/test_communities_screens_overview.py#L15
I think it is more than 1? Or if it is 1 test case, maybe rename it saying it checks all the screens?

File: gui/components/community/create_community_popups.py [hunk 2]
```diff
@@ -162,15 +198,16 @@ def open_next_form(self):
         self._next_button.click()
 
     @allure.step('Create community without file upload dialog usage')
-    def create_community(self, kwargs):
```
Comment (anastasiyaig) [inline gui/components/community/create_community_popups.py:165]: 👍

**Review by anastasiyaig — COMMENTED** [review]
Maybe we can even squeeze the tests for create-edit-delete channel and create-edit-delete categories in 2 scripts?

**Comment by Valentina1133** [comment]
> Maybe we can even squeeze the tests for create-edit-delete channel and create-edit-delete categories in 2 scripts?

I worked only on create community and edit community tests here. I am working on categories now (branch chore/update_community_categories_tests) and already replaced several tests by one add_edit_remove and the same for member role test. Will work on community channel later.

**Comment by anastasiyaig** [comment]
ah okay, i was super optimistic :) Sure, thank you

File: tests/communities/test_communities_screens_overview.py (hunk 1 above)
Comment (Valentina1133) [reply tests/communities/test_communities_screens_overview.py:20]: Added more references

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**